### ISSUE TITLE
Bug: language interests are not accessible on the user settings page

### ISSUE THREAD

**Issue by nickytonline** [open-sauced/app#2366]
1. Ensure you're logged in
2. Go to the settings page
3. Use the tab key to navigate the form. When you get to the interests section, notice that if you tab, it skips all interests and gives focus to the Update Interests button.

https://github.com/open-sauced/app/assets/833231/be28dc53-71a4-4ff1-9ca2-48b1cf315f19
